images/BUILD

```diff
@@ -0,0 +1,61 @@
+package(default_visibility = ["//visibility:public"])
+
+load(
+    "@package_bundle//file:packages.bzl",
+    "packages",
+)
+
+load(
+    "@io_bazel_rules_docker//container:container.bzl",
+    "container_image",
+    "container_push",
+)
+
+container_image(
+    name = "build-base",
+    base = "@base//image",
+    debs = [
+        packages["openssh-client"],
+        packages["git"],
+    ],
+)
+
+container_image(
+    name = "build-base-debug",
+    base = "@base-debug//image",
+    debs = [
+        packages["openssh-client"],
```

Comment:
Are packages like `libedit2` necessary for `openssh-client`? Can you add a comment to that effect?

Reply:
There's a comment a link to the package in `WORKSPACE` that shows required deps

Reply:
I'll mention go to `WORKSPACE` for comments